src/renderer/components/table/table.tsx

```diff
@@ -1,19 +1,17 @@
 import "./table.scss";
 
 import React from "react";
+import lodash from "lodash";
```

Comment:
Unless you have changed all the imports of this in the project, I think that we should try and be consistent.

```suggestion
import _ from "lodash";
```

Reply:
Offtopic for this PR, but I think we should import individual functions from lodash (or even lodash sub packages, not sure if tree shaking works better with that).

Reply:
FYI: tree-shaking doesn't work with `webpack` AFAIK..